packages/vue/src/components/list/SingleList.jsx

```diff
@@ -262,10 +266,9 @@ const SingleList = {
 		setValue(nextValue, props = this.$props) {
 			let value = nextValue;
 
-			if (nextValue === this.$data.currentValue) {
+			if (nextValue === this.$data.currentValue && !props.enableStrictSelection) {
```

Comment:
@anik-ghosh-au7 What is this change about? We should be using the similar approach as react.

Reply:
@bietkul I have updated the changes